src/app/tenant/sunbird/css/custom.css

```diff
@@ -1168,7 +1168,7 @@ h1, h2, h3, h4, h5,
 }
 .footer a{
     color: #fff !important;
-    text-decoration: none;
+    /* text-decoration: none; */
```

Comment:
@Ekta29yadav These changes move into the tenant repo- https://github.com/NIUANULP/nulp-tenant